cmd/kubectl-direct_csi/volumes_list.go

```diff
@@ -150,6 +165,8 @@ func listVolumes(ctx context.Context, args []string) error {
 			emptyOrBytes(v.Status.TotalCapacity),  //CAPACITY
 			v.Status.NodeName,                     //SERVER
 			driveName(drivePaths[v.Status.Drive]), //DRIVE
+			v.ObjectMeta.Labels[directcsi.Group+"/pod.name"],
```

Comment:
Incase if the value is empty - "" , Can we show an hiphen instead? Like we do for the drives listing (showing total capacity and allocated capacity) 

```
cleanLabel := func(val string) string {
	if val == "" {
		return "-"
	}
	return val
}
```

Reply:
Done